bramkouwen/Neatly-In-Motion
# Creating Contact Page

## What does this PR do?
* It creates contact.html 
* It creates the basic page layout for contact.html

## How do you check this PR?
* Pull down the branch
* Check the added contact.html file  

## Screenshots 
None.
